### ISSUE TITLE
Videochat integration

### ISSUE THREAD

**Issue by klues** [asterics/AsTeRICS-Grid#36]
Idea: Videochat integration using WebRTC, see https://webrtc.org/start/

**Comment by sabicalija** [comment]
Great idea. :+1:

Have you already thought about a SIP server? Will we use a private one, at our office, or will we use an external provider?

**Comment by klues** [comment]
No, it was just an idea, we didn't think about it in more detail and it's questionable when/if I'll have resources for implementation.

**Comment by deinhofer** [comment]
Maybe use an open video conferencing platform like jitis?
https://github.com/jitsi/jitsi-meet/blob/master/doc/api.md
